docs/configuration.md

```diff
@@ -9,6 +9,35 @@ menu:
     weight: 40
 ---
 
+# Default Directories
+
+Poetry uses these following default directories
+
+## Config Directory
+
+- Linux: `$XDG_CONFIG_HOME/pypoetry` or `~/.config/pypoetry`
+- Windows: `%APPDATA%\pypoetry`
+- MacOS: `~/Library/Preferences/pypoetry`
+
+You can override this Config directory by setting `POETRY_CONFIG_DIR` environment variable
+
+
+## Data Directory
+
+- Linux: `$XDG_DATA_HOME/pypoetry` or `~/.local/share/pypoetry`
+- Windows: `%APPDATA%\pypoetry`
+- MacOS: `~/Library/Application Support/pypoetry`
+
+You can override this Data Directory by setting `POETRY_DATA_DIR` environment variable. If `POETRY_HOME` environment variable is available, then `POETRY_HOME` will be given higher priority.
+
+## Cache Directory
+
+- Linux: `$XDG_CACHE_HOME/pypoetry` or `~/.cache/pypoetry`
+- Windows: `%APPDATA%\pypoetry\Cache`
+- MacOS: `~/Library/Caches/pypoetry`
+
+You can override this Cache directory by setting `POETRY_CACHE_DIR` environment variable
+
 # Configuration
```

Comment:
This is now not rendered (as this is second h1 on one page). What's more, text after this header should be reworked as this doubles the information about config directories.

Reply:
@Secrus I reworked and put "Default Directories" section under "Configuration" header. Hope this will be proper now.